lucien-ios/Dashboard/LendingCollectionViewController.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LendingViewController.swift
+//  lucien-ios
+//
+//  Created by Fang, Gracie on 10/9/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import UIKit
+
+final class LendingCollectionViewController: UIViewController, UICollectionViewDataSource, UICollectionViewDelegate, UICollectionViewDelegateFlowLayout {
+
+    @IBOutlet weak var collectionView: UICollectionView!
+
+    let reuseIdentifier = "comicCell"
+    let objects = ["Princess", "Gracie", "Kittens"]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.collectionView.register(UINib(nibName:"LendingCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: reuseIdentifier)
```

Comment:
Rather than hardcoding in the nib name, use `String(describing: LendingCollectionViewCell.self)`. The function gets the name of the class dynamically so if you ever need to change the cell class things will stay in sync